.github/actions/frontend/runtime/validate/action.yml

```diff
@@ -6,9 +6,20 @@ name: Runs validation on the repo
 runs:
   using: "composite"
   steps:
+    - name: check_for_auth_token
+      shell: bash
+      if: ${{ env.NODE_AUTH_TOKEN == '' }}
+      run: |
+        echo "the secret \"NODE_AUTH_TOKEN\" has not been set"
+        exit 1
     - name: setup_env
       if: ${{ !fromJSON(env.TEST) }}
       uses: ./.github/actions/frontend/setup_env
+      with:
+        use_dev: true
+    - name: install_remote_types
+      run: npm run postinstall --if-present && npm run dependencies --if-present
```

Comment:
B/c we `--ignore-scripts` in `setup_env`, we have to ensure remote types get installed by manually calling `postinstall` or `dependencies`